### PR TITLE
[process] Removing STF from the environment

### PR DESCRIPTION
**Pull request by leifmadsen**

### CODE REVIEW

**Comment by leifmadsen** [comment]
@JoanneOFlynn2018 once you review and approve, feel free to merge. If you have suggested changes, leave them here and I'll commit then merge after. Thanks!

File: doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+To remove {ProjectShort} from an {OpenShiftShort} environment, complete the following tasks:
+
+. Delete the namespace. For more information, see <<removing-the-namespace>>.
+. Delete the OperatorSource. For more information, see <<removing-the-operatorsource>>.
+
+[id='removing-the-namespace']
+== Removing the namespace
+
+Most of the deployed resources for {ProjectShort} can be removed from {OpenShiftShort} by deleting the namespace. To delete a namespace run the command:
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc:34]:
```suggestion
To remove most of the deployed resources for {ProjectShort} from {OpenShiftShort}, delete the namespace:
```

File: doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+[source,bash]
+----
+oc delete project service-telemetry
+----
+
+You can validate that all the resources have been removed from the namespace by running `oc get all`.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc:41]:
```suggestion
Validate that the resources have been removed from the namespace:
```

Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc:41]: Separating instruction from command. It's such a short step that it looks better this way, we avoid repetition and avoid using "by running"

File: doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc (hunk 1 above)
Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc:34]: More instructional, replacing "by deleting" with just "delete"

File: doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+[source,bash]
+----
+$ oc get all
+No resources found.
+----
+
+[id='removing-the-operatorsource']
+== Removing the OperatorSource
+
+If future installations of {Project} are no longer expected, the OperatorSource can be removed. Removing the OperatorSource will result in the PackageManifests
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc:52]:
```suggestion
If you do not expect to install {Project} again, remove the OperatorSource. When you remove the OperatorSource, PackageManifests related to {ProjectShort} are removed from the Operator Lifecycle Manager catalog.
```

Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc:52]: So the PackageManifests are automatically removed? You don't have to run a command or anything, correct?

File: doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+related to {ProjectShort} being removed from the Operator Lifecycle Manager catalog.
+
+[source,bash]
+----
+$ oc delete --namespace=openshift-marketplace operatorsource infrawatch-operators
+operatorsource.operators.coreos.com "infrawatch-operators" deleted
+----
+
+You can validate that the {ProjectShort} PackageManifests are removed from the platform by running `oc get packagemanifests | grep InfraWatch`, which is expected to return no result.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc:61]:
```suggestion
Validate that the {ProjectShort} PackageManifests are removed from the platform. If successful, the following command returns no result: 
```

File: doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc (hunk 3 above)
Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc:52]: Correct. When you remove the `CatalogSource` or `OperatorSource` the `PackageManifests` for that source are automatically removed.

**Pull request merged**